IPs/RSKIP428.md

```diff
@@ -0,0 +1,97 @@
+---
+rskip: 428
+title: HSM Segwit Support
+created: 23-APR-24
+author: NC
+purpose: Sca
+layer: Core 
+complexity: 2
+status: Draft
+description: 
+---
+
+|RSKIP          | 428                |
+| :------------ |:-------------------|
+|**Title**      | Support HSM Segwit |
+|**Created**    | 23-APR-24          |
+|**Author**     | NC                 |
+|**Purpose**    | Sca                |
+|**Layer**      | Core               |
+|**Complexity** | 2                  |
+|**Status**     | Draft              |
+
+## Abstract
+
+This RSKIP proposes creating a new event that stores the outpoint values of the inputs used in a 
+peg-out to enable HSM Segwit support in RSK.
+
+## Motivation
+
+To support HSM Segwit, the PowPeg must provide the outpoint values for signing a segwit peg-out[1].
+Currently, selected UTXOs for a peg-out are removed from the available utxos set once 
+the peg-out is created. Storing the outpoint values in an event will allow the PowPeg to retrieve 
+and provide the outpoint values to the HSM, so being able to sign segwit peg-out properly.
+
+## Specification
+
+The proposal is to create a new event to store the Bitcoin Transaction Hash of the peg-out and 
+the outpoint values of each utxo used for it. This event should be emitted whenever a peg-out is 
```

Comment:
Maybe instead of `peg-out` call it `a transaction that spends from the PowPeg address`? Or something similar. Sounds a bit strange to say that a peg-ut could be a migration or a refund transaction